azkaban-web-server/src/main/java/cloudflow/readme.md

```diff
@@ -28,3 +28,16 @@ Sample body:
     "gsalia"
   ]
 }
+
+Get executions filtered by Project Id and User.
```

Comment:
I feel it would be better to create separate files for each endpoint - one for /spaces, one for /executions, etc. That will make it easier to maintain and search.

Reply:
Having a single Readme makes it easier to quickly compare if the request and responses for the different APIs have some conformity. Ideally this should be covered in some unit tests which could be in different files but till then I would prefer a single file. We can discuss further though.

Reply:
You are right - we will be writing unit tests soon. I don't feel strongly about this so let's leave it as it is.